### PR TITLE
add test suite with jasmine task #41

### PR DESCRIPTION
**Pull request by blackout314**
task #41 
- test suite is first thing we need for stop to develop shit

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#56](https://github.com/zold-io/zold.github.io/issues/56); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Thu Jun 28 2018 14:36:57 GMT+0200 (CEST)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
```
Comment (yegor256) [inline karma.conf.js]: @blackout314 what's up with this file? Copied it from somewhere? Let's format it right and remove comments and empty lines.

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
+
+
+    // web server port
+    port: 9876,
```
Comment (yegor256) [inline karma.conf.js]: @blackout314 what if this port is not available on the server? The test will crash?

**Comment by yegor256** [comment]
@blackout314 see above. Aside from that, why do we need this `npm-shrink.json` file in the repo? It looks like a temporary file to me.

File: package.json [hunk 3]
```diff
@@ -19,10 +20,17 @@
...
     "grunt-replace": "^1.0.1",
     "grunt-sass": "2.1.0",
-    "grunt-sass-lint": "0.2.2"
+    "grunt-sass-lint": "0.2.2",
+    "jasmine-core": "^2.7.0",
+    "karma": "0.13.21",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-jasmine": "0.3.8",
+    "karma-mocha-reporter": "~1.1.1",
+    "karma-phantomjs-launcher": "^1.0.4"
```
[the remark this answers is not in the document]
Comment (blackout314) [reply package.json]: @yegor256 yes we need all because phantomjs is engine for travis and chrome for local develop

**Comment by blackout314** [comment]
@yegor256 npm-shrink is necessary for force right version of broken package (security issue)
remember?

File: karma.conf.js (hunk 2 above)
Comment (blackout314) [reply karma.conf.js]: @yegor256 is local port, not needed by remote

File: karma.conf.js (hunk 1 above)
Comment (blackout314) [reply karma.conf.js]: @yegor256 is generated from `karma init`

**Comment by blackout314** [comment]
@yegor256 you waste my time
- dont remember past ticket (about security, i add npm-shrink for this, and in this ticket you said is temporany file) and this **waste** my time
- dont know test suite for javascript, i mean karma, all javascript guy know standard file generated from `karma init` and you comment it **waste** my time

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#56](https://github.com/zold-io/zold.github.io/issues/56) is already in scope

**Comment by 0crat** [comment]
Job `gh:zold-io/zold.github.io#56` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#56](https://github.com/zold-io/zold.github.io/issues/56) is now out of scope

**Comment by 0crat** [comment]
Job `gh:zold-io/zold.github.io#56` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order

**Pull request closed without merging**